src/main.js

```diff
@@ -244,3 +244,81 @@ let destinations = [
   ]
 
 
+  //visited and bucktelist lists
+  const visited = destinations.filter(item => item.visited === true);
+  const bucketlist = destinations.filter(item => item.visited === false);
+ 
+
+  //get list of urls
+  const getURL = (destinations) => {
+    let url= destinations.map(obj => obj.photo.valueOf());
+    return url;
+  }
+
+  //create elements for each city
+  const elements= destinations.forEach(destination => {
+
+    //picture element
+    const createImg = (url) => {
+      let figure = document.createElement('figure');
+    //img element
+      let img = document.createElement('img');
+      img.style.objectFit ='cover';
+      document.body.appendChild(figure);
+      figure.appendChild(img);
+    //img caption
+      let figcaption= document.createElement('figcaption');
+      figure.appendChild(figcaption);
+      figcaption.innerText= `${destinations.city}, ${destinations.country}`
+
+
+      //get urls
+      for(let i=0; i<destinations.length; i++) {
+        img.src = url[i];
+      };
```

Comment:
This loop is not needed (and wrong). You can do directly `img.src = destination.photo`. What made u think you needed to do a loop here?

Reply:
The idea was: I have the variable ```url```(a list of all urls) so I wanted to loop through ```destinations``` and for each object in an array assign the ```img.scr```  the correct link, it should get the index of each element that goes through the loop. I get that it's wrong, just not why?

Reply:
`url[i]`, does indeed get the link correctly. (If you console.log it inside the loop, you'll see it)
But you are assigning each url to the same thing `img.src`, overriding it over and over again with different URLs. In the end, the result is the URL of the last index of the loop.

Reply:
Ohhh now I see. So... I need a new instance of img.src for each city? It's something I remember in relation to classes, does it make sense in this context? Or another loop?

Reply:
Maybe you are still confused (or I didn't understand your goal).
In this scope, we just need to worry about one image per destination, right?. We are already within the loop (line 260) and in this specific destination details. So we do not need to do a new loop. Does it make sense?

Reply:
Wow I really overcomplicated there 😄  ```img.src = destination.photo``` worked like a charm 🙌🏻